src/components/cl/ucc_cl.c

```diff
@@ -12,6 +12,9 @@ ucc_config_field_t ucc_cl_lib_config_table[] = {
     {"", "", NULL, ucc_offsetof(ucc_cl_lib_config_t, super),
      UCC_CONFIG_TYPE_TABLE(ucc_base_config_table)},
 
+    {"TLS", "all", NULL, ucc_offsetof(ucc_cl_lib_config_t, tls),
```

Comment:
is there a way for users to know possible TLS values? can we have help string instead of NULL?

Reply:
Added the doc string and available TLS. Plz see how it is done: first i set doc string in config field to NULL. Then on the first call to cl_lib_parser i construct the tls list string (at that moment the TL framework is loaded and i can build the list) and then i reset the "doc" pointer to the constructed message with tls list.
The result ucc_info output:
```
#
# List of TLs used by a given CL component.
# Allowed values: either "all" or comma-separated list of: nccl,ucp
#
# syntax:    comma-separated list of: string
# inherits:  UCC_TLS
#
UCC_CL_BASIC_TLS=all

```